### DISCUSSION TITLE
Request: AppImage

### DISCUSSION THREAD

**Discussion by mj162** [gonzaarcr/Fildem#76] [Ideas]
Thank you for Fildem,

We use HP ThinPro thin clients.  Installing anything on flash would invalidate support, so all our third-party apps are installed on an NFS share or USB stick.  This is only practicable and supportable for AppImages.  Please consider releasing Fildem as an executable AppImage.  Thank you.

**Comment by y1ca1** [comment]
Please consider it!

**Comment by aargar1** [comment]
It is impossible to pack something like Fildem as an AppImage.

**Comment by mj162** [comment]
Did the AppImage people say that they'd seek to amend the
tools so you could do in future, or that they'd no plans
to make them be able to pack things like filedem?

**Reply by aargar1** [reply]
No... because of the nature of Fildem, AppImages are meant to package stand-alone apps, not apps that integrate with the system. For example, fildem won't be able to access the menu of different apps in order to show it. Also, fildem has a gnome extension, which must be stored on the main drive.

**Comment by mj162** [comment]
The call for AppImage was just a means to an end not an end in itself.

If Filedem can be run from a share or removable media, I'm all ears.
